src/core/IconButton/index.stories.tsx

```diff
@@ -1,6 +1,7 @@
 import { Args, Story } from "@storybook/react";
 import React from "react";
 import Icon from "../Icon";
+import { getSpaces } from "../styles";
```

Comment:
`getSpaces` needs `theme` as a parameter and since we don't have theme object in storybook stories, I think we can directly import `defaultTheme` object and use it to get `spaces`.

```
import { defaultAppTheme } from "../styles";
```